Bottom: eed4486d2c0ae48fe9844c222ab89a5a5756f077
Top:    d6a62ad1dead7e506de4197029a2612dde20961f
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-17 16:19:12 +0800

Refresh of vhost_net-introduce-tx

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 2fc4f4b..ef228fe 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -487,7 +487,6 @@ static bool vhost_exceeds_weight(int pkts, int total_len)
 
 static int get_tx_bufs(struct vhost_net *net,
 		       struct vhost_net_virtqueue *nvq,
-		       struct iovec iov[], unsigned int iov_size,
 		       struct msghdr *msg,
 		       unsigned int *out, unsigned int *in,
 		       size_t *len, bool *busyloop_intr)
@@ -567,8 +566,8 @@ static void handle_tx(struct vhost_net *net)
 			vhost_zerocopy_signal_used(net, vq);
 
 		busyloop_intr = false;
-		head = get_tx_bufs(net, nvq, vq->iov, ARRAY_SIZE(vq->iov),
-				   &msg, &out, &in, &len, &busyloop_intr);
+		head = get_tx_bufs(net, nvq, &msg, &out, &in, &len,
+				   &busyloop_intr);
 		/* On error, stop handling until the next kick. */
 		if (unlikely(head < 0))
 			break;
